typedoc.json

```diff
@@ -1,6 +1,6 @@
 {
   "extends": ["./typedoc.base.json"],
-  "entryPoints": ["packages/pre", "packages/shared", "packages/taco"],
+  "entryPoints": ["packages/pre", "packages/shared", "packages/taco", "packages/taco-auth"],
```

Comment:
Should we mark `packages/pre` as deprecated or remove it altogether from the docs?

Reply:
✔️ - good idea.